### PR TITLE
Update pr-commands.yaml workflow

### PR DESCRIPTION
**Pull request by maurolepore**
Relates to [AB#3100](https://dev.azure.com/2DegreesInvesting/a5201d6a-87ad-4acf-b862-9cba0e1ae681/_workitems/edit/3100)

This PR updates a workflow that allows us to run an automated styler on a PR. That means the contributor doesn't need to know how to do that locally.

### `?use_github_action_pr_commands()`

> This workflow enables the use of two R-specific commands in pull request issue comments:
* `/document` to run `roxygen2::roxygenise()` and update the PR
* `/style` to run `styler::style_pkg()` and update the PR

### CODE REVIEW

File: .github/workflows/pr-commands.yaml [hunk 1]
```diff
@@ -1,55 +1,75 @@
...
+
 name: Commands
+
 jobs:
   document:
-    if: startsWith(github.event.comment.body, '/document')
+    if: ${{ github.event.issue.pull_request && (github.event.comment.author_association == 'MEMBER' || github.event.comment.author_association == 'OWNER') && startsWith(github.event.comment.body, '/document') }}
     name: document
-    runs-on: macOS-latest
+    runs-on: ubuntu-latest
```
Comment (MirijaHa) [inline .github/workflows/pr-commands.yaml:13]: out of interest, why the change of os?

Comment (maurolepore) [reply .github/workflows/pr-commands.yaml:13]: (Sorry -- I responded this before but apparently I did not hit "Comment".)

I don't know. I brainlessly copied the r-lib workflow via `usethis::use_github_action_pr_commands()`.

**Comment by maurolepore** [comment]
Oh, wierd! Only now I realize this PR touches more files than I expected, i.e. just .github/workflows/pr-commands.yaml. I'll fix that now.

**Comment by maurolepore** [comment]
Replaced by #263 as this PR accidentally got way more complex than it should.
